src/components/SearchResults.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/SearchResults.css";
+
+const SearchResults = ({ results }) => {
+  return (
+    <div className="SearchResults" data-testid="SearchResults">
+        <img className="card-image" data-testid="card-image" src={results} alt="spaceImage" />
```

Comment:
I would do a map over the results here and place each url in the map into an image tag src property.